src/video-player.js

```diff
@@ -44,6 +44,15 @@ const PLAYER_PARAMS = CLOUDINARY_PARAMS.concat([
   'textTracks'
 ]);
 
+const TEST_URL_DEFAULT_REQUEST = { method: 'head' };
+const TEST_URL_GET_REQUEST = { method: 'get', withCredentials: true, headers: { Range: 'bytes=0-0' } };
+
+const getTestUrlRequest = (uri, options = {}) => {
```

Comment:
I would send into this method only the relevant option (options.testUrlMethod).

Reply:
Good idea

Reply:
done